packages/babel-cli/test/fixtures/babel/env - SHOW_CONFIG_FOR/stdout.txt

```diff
@@ -125,3 +124,4 @@ programmatic options from @babel/cli
   },
   "filename": "./src/index.js"
 }
+"use strict";
```

Comment:
Maybe we should print a separator between the code and the config? Something like
```

================================================================================

```

Reply:
I don't think the debug output should be directed to `stderr`. 

For example `curl -v http://example.com/example.avif > example.avif` will also break, unless we specify `-O example.avif`. Similary in this case, consider use `--out-file example.js` instead.